packages/react/src/hooks/registration.ts

```diff
@@ -0,0 +1,34 @@
+import { useMemo } from 'react'
+
+import { useSelector } from '@xstate/react'
+
+import { useAuthenticated, useNhostInterpreter, useReady } from './common'
+
+export const useSignUpEmailPassword = (stateEmail?: string, statePassword?: string) => {
```

Comment:
To maintain consistency here, we should name this hook as `useEmailPasswordSignUp` instead, since `useSignIn` hooks are following this pattern.